test/io/GeantImporter.test.cc

```diff
@@ -71,3 +92,79 @@ TEST_F(GeantImporterTest, import_particles)
     EXPECT_VEC_EQ(expected_loaded_names, loaded_names);
     EXPECT_VEC_EQ(expected_loaded_pdgs, loaded_pdgs);
 }
+
+//---------------------------------------------------------------------------//
+TEST_F(GeantImporterTest, import_tables)
+{
+    GeantImporter import(root_filename_.c_str());
+    auto          data = import();
+
+    EXPECT_GE(data.physics_tables->size(), 0);
+
+    bool lambda_kn_gamma_table = false;
+
+    for (auto table : *data.physics_tables)
+    {
+        EXPECT_GE(table.physics_vectors.size(), 0);
+
+        if (table.particle == PDGNumber{celeritas::pdg::gamma()}
+            && table.table_type == GeantTableType::Lambda
+            && table.process == GeantProcess::compt
+            && table.model == GeantModel::KleinNishina)
+        {
+            lambda_kn_gamma_table = true;
+            break;
+        }
+    }
+    EXPECT_TRUE(lambda_kn_gamma_table);
+}
+
+//---------------------------------------------------------------------------//
+TEST_F(GeantImporterTest, import_geometry)
+{
+    GeantImporter import(root_filename_.c_str());
+    auto          data = import();
+
+    auto map = data.geometry->volid_to_matid_map();
+    EXPECT_EQ(map.size(), 4257);
+
+    // Fetch a given GeantVolume provided a vol_id
+    GeantGeometryMap::vol_id volid  = 10;
+    GeantVolume              volume = data.geometry->get_volume(volid);
+    EXPECT_EQ(volume.name, "TrackerPatchPannel");
+
+    // Fetch respective mat_id and GeantMaterial from the given vol_id
+    auto          matid    = data.geometry->get_matid(volid);
+    GeantMaterial material = data.geometry->get_material(matid);
+
+    // Material
+    EXPECT_EQ(matid, 31);
+    EXPECT_EQ(material.name, "Air");
+    EXPECT_EQ(material.state, MaterialState::gas);
+    EXPECT_SOFT_EQ(material.temperature, 293.15);
+    EXPECT_SOFT_EQ(material.density, 0.00121399936124299);
+    EXPECT_SOFT_EQ(material.electron_density, 3.6523656201748414e+20);
+    EXPECT_SOFT_EQ(material.atomic_density, 5.0756589772243755e+19);
+    EXPECT_SOFT_EQ(material.radiation_length, 30152.065419629631);
+    EXPECT_SOFT_EQ(material.nuclear_int_length, 70408.106699294673);
```

Comment:
You should add units to all these in case we change them later.

Reply:
Done.